.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 14.17.3
```

Comment:
We're using nvm to manage the Node.js version, so this is a duplication of functionality that needs to be justified, and at least synchronised with the other [config file](https://github.com/input-output-hk/cardano-js-sdk/blob/2d92462a8809ac6deca837a70425a53873130235/.nvmrc#L1)

Reply:
🆗